gradle/versions.gradle

```diff
@@ -36,7 +36,7 @@ ext {
     mockitoAndroidVersion = "3.6.28"
     multidexVersion = "2.0.1"
     nimbusVersion = "8.2"
-    powerMockVersion = "1.6.6"
+    powerMockVersion = "2.0.9"
```

Comment:
Can we update in other places such as android-complete as well?

Reply:
I think android-complete is updated.

Reply:
nope/